mobify/platform-scaffold
# [Integration manager] checkout urls

This PR moves the `cartURL` into the app store, and updates all previous references to `/checkout/cart/` to pull the `cartURL` off the state instead. This work is required before we start work on the demandware cart connector

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1271
 **Linked PRs**: n/a

## Changes
- Renamed `checkoutURL` to more accurate `checkoutShippingURL`
- Add `cartURL` to the app state
- Update app to pull `cartURL` from the state (includes: PDP, mini-cart, checkout-shipping and checkout-payment) 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- confirm the PDP, mini-cart, checkout-shipping and checkout-payment pages all work the same as they did before
